wgpu-native/src/conv.rs

```diff
@@ -219,50 +222,46 @@ pub fn map_depth_stencil_state_descriptor(
         depth: if desc.depth_write_enabled
             || desc.depth_compare != resource::CompareFunction::Always
         {
-            hal::pso::DepthTest::On {
+            Some(hal::pso::DepthTest {
                 fun: map_compare_function(desc.depth_compare),
                 write: desc.depth_write_enabled,
-            }
+            })
         } else {
-            hal::pso::DepthTest::Off
+            None
         },
         depth_bounds: false, // TODO
         stencil: if desc.stencil_read_mask != !0
             || desc.stencil_write_mask != !0
             || desc.stencil_front != pipeline::StencilStateFaceDescriptor::IGNORE
             || desc.stencil_back != pipeline::StencilStateFaceDescriptor::IGNORE
         {
-            hal::pso::StencilTest::On {
-                front: map_stencil_face(
-                    &desc.stencil_front,
-                    desc.stencil_read_mask,
-                    desc.stencil_write_mask,
-                ),
-                back: map_stencil_face(
-                    &desc.stencil_back,
-                    desc.stencil_read_mask,
-                    desc.stencil_write_mask,
```

Comment:
Looks like the values gone from StencilFace popped up here..?
Should I be setting these values to the same values that were previously used in StencilFace?

Reply:
I think what you are doing right now (just setting the stencil to `Dynamic`) is correct